config/routes.rb

```diff
@@ -447,6 +447,7 @@
 
   get "stylesheets/:name.css.map" => "stylesheets#show_source_map", constraints: { name: /[-a-z0-9_]+/ }
   get "stylesheets/:name.css" => "stylesheets#show", constraints: { name: /[-a-z0-9_]+/ }
+  get "javascripts/:digest.js" => "javascripts#show", constraints: { digest: /\h{40}/ }
```

Comment:
tightened this significantly by only allowing 40 hexdigit chars, [and added tests](https://github.com/discourse/discourse/pull/6447/files#diff-698a1e4385463ecf5c8eac5c8c65b9feR17).

Reply:
Excellent thanks!